.changeset/old-yaks-grin.md

```diff
@@ -0,0 +1,5 @@
+---
+'@twind/preset-tailwind': patch
```

Comment:
Oops, sorry for that! I mixed up the affected packages

Reply:
No worries. Thank you for contributing!